src/menu.rb

```diff
@@ -0,0 +1,175 @@
+require './book'
+require './rental'
+require './student'
+require './teacher'
+require './persons'
```

Comment:
- [ ] This is flagging some error on my end. A quick fix for the error is to change `require './file'` to `require_relative 'file'`. DO this in all your files and the error will be solved.
![image](https://user-images.githubusercontent.com/94551484/216785805-173f7ba0-390e-4b67-b648-63dd15ddcbba.png)